README.md

```diff
@@ -123,13 +123,26 @@ window.setTimeout(function () {
 }, splashDuration - fadeDuration);
 ```
 
-### iOS Quirks
+### iOS and Windows Quirks
 
 - `ShowSplashScreenSpinner` (boolean, defaults to `true`): Set to `false`
   to hide the splash-screen spinner.
 
         <preference name="ShowSplashScreenSpinner" value="false"/>
 
+### Windows Quirks
+
+- `SplashScreenSpinnerColor` (string, defaults to system accent color): hash, rgb notation or CSS color name.
+
+        <preference name="SplashScreenSpinnerColor" value="#242424"/>
+        <preference name="SplashScreenSpinnerColor" value="DarkRed"/>
+        <preference name="SplashScreenSpinnerColor" value="rgb(50,128,128)"/>
+
+- `SplashScreenBackgroundColor` (string, defaults to #464646): hex notation.
+
+        <preference name="SplashScreenBackgroundColor" value="0xFFFFFFFF"/>
+
+
```

Comment:
Shouldn't it be mentioned to add "Scale 100", "Scale 140" and "Scale 180" images? (620 x 300 px, 868 x 420 px and 1116 x 540 px, https://msdn.microsoft.com/en-us/library/windows/apps/mt187306.aspx)